### DISCUSSION TITLE
Netbox benchmarking results

### DISCUSSION THREAD

**Discussion by kdambekar** [netbox-community/netbox#10328] [Q&A]
Hi, 

Are any benchmarking/performance testing results available for Netbox community edition ? I am more interested in IPAM module.
Thank you.

**Comment by hagbarddenstore** [comment]
Benchmarking what exactly?

**Reply by kdambekar** [reply]
Hi,

I am looking for details like- 

- number of IP addresses, prefixes supported by the IPAM module
- Typical REST API response time for fairly loaded inventory
- Frontend UI response time
- Hardware sizing guidelines

**Reply by jeremystretch** [reply]
There's no limit on the number of objects that can be created in the database, for any object type. The other items you've listed are subjective and will vary widely depending on myriad factors, such as allocated resources and use patterns.
